docs/source/examples/spot-jobs.rst

```diff
@@ -175,3 +202,17 @@ Here are some commands for managed spot jobs. Check :code:`sky spot --help` for
     # Cancel a spot job by name
     $ sky spot cancel -n bert-qa
 
+
+Advanced note (spot controller)
+-------------------------------
+
+There will be a single spot controller cluster (a small on-demand CPU VM) running in the background to manage all the spot jobs.
+It will be autostopped after 30 minutes of inactivity. Normally, **no user intervention** is needed. 
+You can find the cluster with :code:`sky status -a`, and refresh the status with :code:`sky status -ar`.
+
+Although, the cost of the stopped spot controller is negligible (less than $0.1 per day), you can still tear it down manually with 
```

Comment:
Should we also mention the default cost of a running controller? (~$0.4/hr)

Reply:
Good point! Added. Thanks!